llvm/test/CodeGen/X86/vector-reduce-fmaximum.ll

```diff
@@ -816,64 +816,6 @@ define float @test_v16f32(<16 x float> %a0) {
 ; SSE41-NEXT:    orps %xmm4, %xmm0
 ; SSE41-NEXT:    retq
 ;
-; AVX-LABEL: test_v16f32:
```

Comment:
Looks like its missing a non-common check-prefix

Reply:
regening, not sure what happened.

Reply:
still not fixed